nnsvs/model.py

```diff
@@ -60,7 +57,6 @@ def __init__(self, in_dim, hidden_dim, out_dim, num_layers=2, dropout=0.0):
         self.last_linear = nn.Linear(hidden_dim, out_dim)
         self.relu = nn.ReLU()
         self.dropout = nn.Dropout(dropout)
-
```

Comment:
Same as above

Reply:
I added new line there in the latest commit.